scss/list.scss

```diff
@@ -0,0 +1,43 @@
+.list {
+  align-items: center;
+  justify-content: center;
+  min-height: 100vh;
+  min-width: 100vw;
+
+  .content {
+    display: flex;
+    flex-flow: row wrap;
+    justify-content: center;
+    max-width: 936px;
+    width: 100%;
+
+    .author {
+      color: $gray;
+      font-size: 14px;
+    }
+    
+    .title {
+      font-size: 16px;
+      font-weight: bold;
+      margin: 0 0 8px 0;
+    }
+
+    .card {
+      flex-flow: column nowrap;
+      margin: 18px;
+      max-height: 300px;
+      max-width: 198px;
+      padding: 28px 28px 23px 28px;
+    }
+
+    .cover {
```

Comment:
this is so much specificity so 

use classes in this way for easier overriding 

.list  {
  ...
}

.content {
  ...
}
.author {
  ...
}

.title {
  ...
}
etc

Reply:
ok